samples/mobile/src/main/java/com/gluonhq/maps/samples/mobile/MobileSample.java

```diff
@@ -68,7 +70,28 @@
 
     @Override
     public void start(Stage stage) {
-        MapView view = new MapView();
+        TileRetriever osm=new CachedOsmTileRetriever() {
+             public String buildImageUrlString(int zoom, long i, long j) {
+                 return "http://tile.openstreetmap.org/" +zoom+"/"+ i + "/" + j + ".png";
+                    
+             }
+             public String copyright() {
+                 return "Map data © OpenStreetMap contributors, CC-BY-SA. Imagery © OpenStreetMap, for demo only.";
+             }
+        };
+        
+        TileRetriever mapBox=new CachedOsmTileRetriever() {
+            public String buildImageUrlString(int zoom, long i, long j) {
+               String url= "https://api.mapbox.com/styles/v1/mapbox/satellite-v9/tiles/256/"+zoom+"/"+i+"/"+j+"?access_token=xxxx";               
```

Comment:
Probably we need some indication to the developer that a token is required and how it can be obtained.

Reply:
I have added some comments (I could not find a full list of supported tiles in mapbox doc though).
Also I have added my own token so you can see how it can work out of the box.
But you will want to replace it by your (Gluon) own token, because I will invalidate the token at some point